src/Markdown.MAML/Renderer/Markdownv2Renderer.cs

```diff
@@ -186,13 +186,13 @@ private void AddParameters(MamlCommand command)
         private void AddCommonParameters()
         {
             AddHeader(ModelTransformerBase.PARAMETERSET_NAME_HEADING_LEVEL, MarkdownStrings.CommonParametersToken, extraNewLine: false);
-            AddParagraphs(MarkdownStrings.CommonParametersText);
+            AddParagraphs(MarkdownStrings.CommonParametersText,false, true);
```

Comment:
nit: `,false,` -> `, false,`

Please, use the named parameters to pass booleans: i.e.  `(noNewLines: false, skipAutoWrap: true)`